### PR TITLE
Translation of the 2020 mobile web chapter to Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
#1750 
Kept mSpeed as is but I can change it if needed

### CODE REVIEW

File: src/content/nl/2020/mobile-web.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+Hoewel elk type item, zoals lettertype, CSS, JavaScript, enz. een belangrijke rol speelt bij de laadprestaties, nemen we afbeeldingen onder de loep.
+
+Het internet blijft evolueren naar pagina's met veel afbeeldingen, met de groei van bandbreedte en de alomtegenwoordigheid van smartphones. En afbeeldingen brengen kosten met zich mee voor de laadprestaties.
+
+Afbeeldingen met een onjuist formaat en niet-geoptimaliseerde afbeeldingen zijn frequente bronnen voor problemen met de afbeeldingsprestaties. Maar liefst 41,20% van de pagina's heeft afbeeldingen met een onjuist formaat.
+
+{{ figure_markup(
+  image="pages-with-properly-sized-images.png",
+  alt="Pagina's met afbeeldingen van het juiste formaat",
+  caption='Pagina's met afbeeldingen van het juiste formaat',
```
Comment (bazzadp) [inline src/content/nl/2020/mobile-web.md:189]: Need to change caption to double quotes. And I can't see any reason for the extra `alt` so can we remove that (in English too if possible):

```suggestion
  caption="Pagina's met afbeeldingen van het juiste formaat",
```

File: src/content/nl/2020/mobile-web.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+##### A/B-testen
+
+A/B-testen is een cruciaal hulpmiddel voor het nemen van datagestuurde beslissingen over ontwerp- en UX-kwesties. A/B-testen maakt het mogelijk om te valideren dat de UX- en ontwerpwijzigingen de beoogde statistieken meetbaar verbeteren en geen onverwachte regressies veroorzaken.
+
+Hier is een greep uit de ontwerpvragen die A/B kunnen worden getest:
+* Zou het veranderen van de kleur van een knop de klikfrequentie verhogen?
+* Zou het vergroten van de grootte van klikdoelen het aantal klikken verhogen?
+* Zou het vervangen van het zoekpictogram door een zoekvak het aantal voltooide zoekopdrachten verhogen?
+
+Volgens <a hreflang="en" href="https://www.thirdpartyweb.today/">thirdpartyweb.today</a>, is <a hreflang="en" href="https: //www.optimizely.com">Optimizely</a> het meest populaire product van derden voor A/B-testen, het wordt gebruikt op meer dan 20.000 pagina's.
```
Comment (bazzadp) [inline src/content/nl/2020/mobile-web.md:357]: Extra space:

```suggestion
Volgens <a hreflang="en" href="https://www.thirdpartyweb.today/">thirdpartyweb.today</a>, is <a hreflang="en" href="https://www.optimizely.com">Optimizely</a> het meest populaire product van derden voor A/B-testen, het wordt gebruikt op meer dan 20.000 pagina's.
```

File: src/content/nl/2020/mobile-web.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+featured_stat_3: 2,6s
+featured_stat_label_3: 75e percentiel LCP voor mobiel
+---
+
+## Inleiding
+Mobiel Internet is het afgelopen decennium explosief gegroeid en is nu de belangrijkste manier waarop veel mensen internet ervaren. Desondanks blijven engagement en online verkoop nog steeds achter bij desktop. In dit hoofdstuk kijken we naar recente trends op het mobiele web en analyseren we waarom gebruikersreis vaak moeilijk te voltooien zijn.
+
+In 2020 is het <a hreflang="en" href="https://www.nytimes.com/interactive/2020/04/07/technology/coronavirus-internet-use.html">internetgebruik</a> enorm gestegen, zowel op mobiel als op desktop, als gevolg van de wereldwijde pandemie. Het aantal bezoeken aan nieuwssites, e-commerce en sociale-mediasites is toegenomen doordat mensen over de hele wereld zich hebben aangepast aan een nieuwe levensstijl met lockdowns en sociale afstand. 2020 was een belangrijk jaar in de geschiedenis, zowel voor internet als voor mobiel gebruik.
+
+### gegevensbronnen
```
Comment (bazzadp) [inline src/content/nl/2020/mobile-web.md:28]: Capital G?

```suggestion
### Gegevensbronnen
```

File: src/content/nl/2020/mobile-web.md [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+Hoewel "conversie" klinkt als een concept dat betrekking heeft op e-commercesites, kan een conversie verwijzen naar een succesvolle gebruikerstransactie, zoals aanmelden voor een muziekstreamingservice, een huurwoning boeken, een recensie schrijven op een reissite, enzovoort.
+
+Volgens Comscore Media Matrix vertegenwoordigt het verkeer van mobiele apparaten 79,6% van de tijd die op Amerikaanse winkelsites wordt doorgebracht, maar slechts 32,3% van de Amerikaanse e-commerceverkopen.
+
+Vergeleken met desktop zijn transacties op mobiele apparaten foutgevoelig en vervelend, aangezien gebruikers persoonlijke informatie moeten invoeren met behulp van kleine toetsenborden en schermformaten. Afrekenstromen moeten eenvoudig en kort zijn om gebruikersfrustratie, of erger nog, verlating te voorkomen. 27% van de gebruikers verlaat het afrekenen vanwege een <a hreflang="en" href="https://www.smashingmagazine.com/2018/08/best-practices-for-mobile-form-design/">"te lang / ingewikkeld afrekenproces"</a>. 35% van de gebruikers verlaat het afrekenen als een winkel <a hreflang="en" href="https://baymard.com/blog/ecommerce-checkout-usability-report-and-benchmark">geen gastafrekenen aanbiedt< / a>.
```
Comment (bazzadp) [inline src/content/nl/2020/mobile-web.md:365]: Remove extra spaces:

```suggestion
Vergeleken met desktop zijn transacties op mobiele apparaten foutgevoelig en vervelend, aangezien gebruikers persoonlijke informatie moeten invoeren met behulp van kleine toetsenborden en schermformaten. Afrekenstromen moeten eenvoudig en kort zijn om gebruikersfrustratie, of erger nog, verlating te voorkomen. 27% van de gebruikers verlaat het afrekenen vanwege een <a hreflang="en" href="https://www.smashingmagazine.com/2018/08/best-practices-for-mobile-form-design/">"te lang / ingewikkeld afrekenproces"</a>. 35% van de gebruikers verlaat het afrekenen als een winkel <a hreflang="en" href="https://baymard.com/blog/ecommerce-checkout-usability-report-and-benchmark">geen gastafrekenen aanbiedt</a>.
```

File: src/content/nl/2020/mobile-web.md [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+
+Lees hier meer over <a hreflang="en" href="https://web.dev/responsive-web-design-basics/">responsive webdesign</a>.
+
+Naast organisch verkeer van zoekmachines, kunnen **Advertentieklikken** een belangrijke bron zijn van bezoekers die naar websites worden gestuurd. Net als bij SEO kan het optimaliseren van advertenties belangrijk zijn voor websites die investeren in en verkeer ontvangen van advertenties.
+
+#### Laadprestaties
+
+De eerste indruk is belangrijk. Het tijdig leveren van pagina-inhoud is van cruciaal belang om te voorkomen dat bezoekers in de steek worden gelaten en gebruikersfrustratie voorkomen. Laadprestaties zijn een belangrijk aspect van de acquisitiefase, slechte laadprestaties leiden ertoe dat gebruikers deze reis verlaten.
+
+Een <a hreflang="en" href="https://web.dev/milliseconds-make-millions/">recente studie</a> toonde aan dat 0,1s mSpeed verbetering de conversieratio's met +8,4% verhoogde voor winkelsites en +10,1% voor reissites.
```
Comment (bazzadp) [inline src/content/nl/2020/mobile-web.md:156]: Let's change to mobile speed (can you change English too?)

```suggestion
Een <a hreflang="en" href="https://web.dev/milliseconds-make-millions/">recente studie</a> toonde aan dat 0,1s mobile speed verbetering de conversieratio's met +8,4% verhoogde voor winkelsites en +10,1% voor reissites.
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Spotted a few issues

**Review by bazzadp — APPROVED** [review]

**Comment by noah-vdv** [comment]
> And I can't see any reason for the extra `alt` so can we remove that (in English too if possible):

So in future cases, if there's an `alt` and a `caption` we just remove the `alt`?

**Comment by bazzadp** [comment]
The `caption` is used as the `alt`, when it's not explicitly provided.

The `caption` is what is displayed on screen (and also what's used by screen readers since we expose that with `aria-labelledby` which overrides the `alt`).

The `alt` is used when image fails to download and also for SEO purposes — hough with a decent filename it's probably a bit redundant to be honest, but we didn't use decent filenames for the 2019 edition (we got better this year).

The reason we have both is to allow them to be different of they need to be. Why would you want a different `alt`? For a few reasons:
1. If the caption contains HTML (e.g. code formatting, or `<span lang="en"></span>` tags). There's some dispute whether an `alt` should have HTML or not, but [I agree with this answer](https://stackoverflow.com/a/18856889/2144578) and think not, but `caption` can have it.
2. Kind of an extension to that, if the caption contains a source reference, then you could remove the link from the alt ([example])(https://almanac.httparchive.org/en/2020/capabilities#fig-1). The mobile web uses them but as plain text ([example](https://almanac.httparchive.org/en/2020/mobile-web#fig-7)). Now you've made me think about this some more, I think it should the best answer is a combination of both - have the plain text source as the `alt`, but then the hyperlinked version as the `caption`.
3. Something else I've not thought of yet :-)

So yes if the `alt` and the `caption` are exactly the same (as they were in a couple of places here), then there is no benefit to having both as the template will automatically copy the `caption` into the `alt` and so I think we should remove to avoid duplication and having to translate the same thing twice.

Make sense?

All this is getting very picky of me btw! But good to have standards.

**Pull request merged**